fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -1890,7 +1997,7 @@ public boolean isDatatableAttachedToEntityDatatableCheck(final String datatableN
                 + " JOIN m_entity_datatable_check edc ON edc.x_registered_table_name = xrt.registered_table_name"
                 + " WHERE edc.x_registered_table_name = '" + datatableName + "'";
         final Long count = this.jdbcTemplate.queryForObject(sql, Long.class);
-        return count > 0;
+        return count != null && count > 0;
```

Comment:
count function can never return null. using `int` is more than fine and no need for null check

Reply:
for me it was a checkstyle warning, if it can not return null, it is fine, but lets be safe